projects/vdk-core/plugins/vdk-plugin-control-cli/src/vdk/plugin/control_cli_plugin/execution_skip.py

```diff
@@ -128,14 +131,31 @@ def _skip_job_if_necessary(
     return None
 
 
+@hookimpl(tryfirst=True)
+def vdk_configure(config_builder: ConfigurationBuilder):
+    config_builder.add(
+        key=VDK_EXECUTION_SKIP_CHECKER_ENABLED,
```

Comment:
Remove the `VDK_` prefix. For configuration key XXX it is picked up from environment variable VDK_XXX or from config.ini file option "xxx"

Reply:
I removed the prefix and split the PR in two.